codenav/src/utils.c

```diff
@@ -19,7 +19,7 @@
  *  along with this program; if not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "utils.h"
```

Comment:
don't remove this, it's the include for the plugin's utils.h, aka this file's header